.github/workflows/ci.yml

```diff
@@ -59,3 +59,6 @@ jobs:
 
       - name: Test
         run: yarn test
+
+      - name: Size limit
+        run: yarn size-limit
```

Comment:
I could see maybe adding a comment response if this fails. This way we get the CI failure and comment feedback. No required imo but something we could improve on if we wanted